crates/forge/src/running.rs

```diff
@@ -67,18 +83,54 @@ pub(crate) fn run_from_test_case(
         footer.iter()
     );
     let (hints_dict, string_to_hint) = build_hints_dict(instructions.clone());
-    let core_cairo_hint_processor = CoreCairoHintProcessor {
-        runner: Some(runner),
-        starknet_state: StarknetState::default(),
-        string_to_hint,
-        run_resources: RunResources::default(),
+
+    let block_context = build_block_context();
+    let account_context = build_transaction_context();
+    let mut context = EntryPointExecutionContext::new(
+        block_context.clone(),
+        account_context,
+        block_context.invoke_tx_max_n_steps.try_into().unwrap(),
+    );
+    let test_selector = String::from("test_case").to_felt252();
```

Comment:
```suggestion
    let test_selector = String::from("TEST_CONTRACT_SELECTOR").to_felt252();
```
to emphasise that it is just a random name and it is here only to get inside an invoke transaction

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/589/commits/b7cb7181ca8021423c974d8385f3b08011e80872